narps/model/model-narps_smdl.json

```
@@ -0,0 +1,123 @@
+{
+  "Name": "NARPS",
+  "Description": "NARPS Analysis model",
+  "BIDSModelVersion": "1.0.0",
+  "Input": {
+    "task": "MGT"
+  },
+  "Nodes": [
+    {
+      "Level": "run",
```

```suggestion
      "Level": "Run",
```

I think the spec has all of those with the first letter in upper-case `Run`, `Subject`, `Dataset`.

---

Fixed.